internal/webapi/server.go

```diff
@@ -76,7 +76,11 @@ func (a *Api) AllStatistics(r *http.Request, l *slog.Logger) (*femto.Response[br
 
 	if data == nil {
 		// dont just return nil, which would not be marshalled properly
-		return &femto.Response[broadcast.Workstations]{Status: http.StatusOK, Body: broadcast.Workstations{}}, nil
+		return &femto.Response[broadcast.Workstations]{
```

Comment:
This could be `femto.Ok()` that already sets the content type to `json` by default.